src/core/proj/qgsprojutils.h

```diff
@@ -169,9 +169,23 @@ class CORE_EXPORT QgsProjUtils
      * from it.
      *
      * If \a crs does not contain a horizontal CRS (i.e. it is a vertical CRS) NULLPTR will be returned.
+     *
+     * \see crsToVerticalCrs()
      */
     static proj_pj_unique_ptr crsToHorizontalCrs( const PJ *crs );
 
+    /**
+     * Given a PROJ crs (which may be a compound or bound crs, or some other type), extract the vertical crs
```

Comment:
The mention of a bound crs is confusing here, because the implementation will always return NULL for a bound CRS